src/components/navigation/navigation.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    selector: 'ux-navigation',
+    templateUrl: './navigation.component.html',
+    encapsulation: ViewEncapsulation.None
```

Comment:
We can remove this line and the associated import. This is only required if we specify a styleUrls property here

Reply:
encapsulation and associated import removed.